src/backend/cgen.c

```diff
@@ -27,6 +27,14 @@
 static char __file__[] = __FILE__;      /* for tassert.h                */
 #include        "tassert.h"
 
+/*********************************
+ */
+CodeBuilder::CodeBuilder(code *c)
+{
+    head = c;
+    pTail = c ? &code_last(c)->next : &head;
```

Comment:
Although I guess it's not actually used from the glue layer?  Still an issue though, and still needs to be fixed for DtBuilder.